ui/components/app/confirm/info/row/row.tsx

```diff
@@ -59,7 +59,7 @@ const TOOLTIP_ICONS = {
 const TOOLTIP_ICON_COLORS = {
   [ConfirmInfoRowVariant.Default]: Color.iconMuted,
   [ConfirmInfoRowVariant.Critical]: Color.errorAlternative,
-  [ConfirmInfoRowVariant.Warning]: Color.warningAlternative,
+  [ConfirmInfoRowVariant.Warning]: Color.warningDefault,
 };
```

Comment:
### After

<img width="1511" alt="Screenshot 2024-06-03 at 2 36 37 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d87017e9-e750-45d1-9078-164b78eaf838">

Reply:
can you show an example with an icon?

Reply:
<img width="262" alt="Screenshot 2024-06-05 at 3 09 04 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/bad9a7e2-0dad-4e2f-bde7-96c6b28c3291">
<img width="187" alt="Screenshot 2024-06-05 at 3 09 12 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/7a25b5f3-b804-41e0-bedd-31b496e09691">

Reply:
thanks!